mvpa2/base/constraints.py

```diff
@@ -10,16 +10,20 @@
 
 # TODO: __str__ / __repr__'s
 
+from builtins import map
+from builtins import str
+from past.builtins import basestring
+from builtins import object
 __docformat__ = 'restructuredtext'
 
 import numpy as np
 
 if __debug__:
     from mvpa2.base import debug
 
+
 class Constraint(object):
     """Base class for input value conversion/validation.
-
```

Comment:
we are following numpy docstrings convention.  So empty line here is (was ;)) on purpose.  See https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#id6